tests/test_model_selection/test_importances.py

```diff
@@ -230,7 +230,9 @@ def test_multi_coefs_stacked(self):
         """
         X, y = load_iris(True)
 
-        viz = FeatureImportances(LogisticRegression(random_state=222), stack=True)
+        viz = FeatureImportances(
+            LogisticRegression(solver="liblinear", random_state=222), stack=True
+        )
```

Comment:
This is failing on Travis/Linux with huge RMSEs (like 89 😬), but fine on my local Mac and in Appveyor CI; currently manually investigating setting more of the params here on the LogisticRegression to see if I can resolve it that way...

Reply:
Looks like specifying `liblinear` worked, which seems a bit weird to me but 🤷‍♀ &mdash; I might need to dig in a bit deeper to the sklearn changelog to figure out what exactly they've changed in LR